test/functional/page_objects/visual_builder_page.ts

```diff
@@ -358,6 +358,22 @@ export class VisualBuilderPageObject extends FtrService {
     return await gaugeCount.getVisibleText();
   }
 
+  public async getGaugeColor(): Promise<string> {
+    await this.visChart.waitForVisualizationRenderingStabilized();
+    const [, gaugeColoredCircle] = await this.find.allByCssSelector(
+      '[data-test-subj="tvbVisGaugeContainer"] circle'
```

Comment:
@VladLasitsa @DziyanaDzeraviankina is it possible to avoid of using css selectors?